modin/_version.py

```diff
@@ -86,7 +102,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
             break
         except EnvironmentError:
             e = sys.exc_info()[1]
-            if e.errno == errno.ENOENT:
+            if e.errno == errno.ENOENT:  # type: ignore
```

Comment:
Why ignore?

Reply:
Using `sys.exc_info()` loses type information, and it seems like `errno` isn't a guaranteed field.